finagle-stats-core/src/main/scala/com/twitter/finagle/stats/NonReentrantReadWriteLock.scala

```diff
@@ -1,8 +1,21 @@
 package com.twitter.finagle.stats
 
 import java.util.concurrent.locks.AbstractQueuedSynchronizer
+import java.util.Properties
+import java.lang.Thread
```

Comment:
Would you please remove these 2 imports since they are not used?

Reply:
I was under the impression java.util.Properties was needed to access System.getProperty("os.arch"), but I'll double check. 

We can remove java.lang.Thread, but its needed for access SpinWait() when Finagle moves on from Java8.

Reply:
As an FYI, we have `fatal_warnings` turned on in pants, so unused imports will fail the build. So either you or us would need to remove it or comment it in order to merge the change.